flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskExecutor.java

```diff
@@ -126,6 +129,9 @@
 	/** The metric registry in the task manager */
 	private final MetricRegistry metricRegistry;
 
+	/** The heartbeat manager for job manager and resource manager in the task manager */
+	private final HeartbeatManagerImpl heartbeatManager;
```

Comment:
Yes, the current **HeartbeatManagerImpl** is enough to use, so I did not wrapper it.